src/components/SortBy/SortBy.jsx

```diff
@@ -1,20 +1,26 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
 import styles from './SortBy.scss';
+import * as constants from '../../constants.json';
+import sort from '../../actions/sort';
 
 const cx = classNames.bind(styles);
 
 class SortBy extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      sortBy: 'rate',
+      sortBy: '',
     };
   }
 
-  handelSort(e, val = 'rate') {
+  handelSort(e, val = constants.NO_SORT) {
```

Comment:
method name misspelling :). We can make this function curried. We pass type of sort to the first call and it returns new function wich could be used as a callback